src/Entity/File.php

```diff
@@ -78,4 +78,47 @@ public function setDimensions(?array $dimensions): void
     {
         $this->dimensions = $dimensions;
     }
+
+    /**
+     * A simple shortcut to the image width.
+     * Similar to `$file->getDimensions()[0]`.
+     *
+     * @return int|null Returns `null` if dimensions array is itself null
+     */
+    public function getWidth(): ?int
+    {
+        return null !== $this->dimensions ? $this->dimensions[0] : null;
+    }
+
+    /**
+     * A simple shortcut to the image height.
+     * Similar to `$file->getDimensions()[1]`.
+     *
+     * @return int|null Returns `null` if dimensions array is itself null
+     */
+    public function getHeight(): ?int
+    {
+        return null !== $this->dimensions ? $this->dimensions[1] : null;
+    }
+
+    /**
+     * Format image dimensions for use with html (to avoid layout shifting).
+     *
+     * Usage in twig template:
+     * ```twig
+     * <img src="..." alt="..." {{ image.htmlDimensions }}>
+     * <!-- Will render: -->
+     * <img src="..." alt="..." width="..." height="...">
+     * ```
+     *
+     * @return string|null Returns `null` if dimensions array is itself null
+     */
+    public function getHtmlDimensions(): ?string
+    {
+        if (null !== $this->dimensions) {
+            return \sprintf('width=%s height=%s', $this->getWidth(), $this->getHeight());
```

Comment:
Missing quotes around placeholders

Reply:
Yes, indeed. In fact it was render a double quote.
But I now understand it was the twig html escape mechanism.
So this require to add the `|raw` filter.

I'll do a fix PR.